Sources/App/Views/UserStoryTemplate.swift

```diff
@@ -0,0 +1,91 @@
+import HTMLKit
+
+struct UserStoryData {
+    let userStory: UserStory
+}
+
+struct UserStoryTemplate: HTMLTemplate {
+    @TemplateValue(UserStoryData.self) var context
+
+    var body: HTML {
+        Document(type: .html5) {
+            Head {
+                Title { context.userStory.name }
+                Author { "Renaud Jenny" }.twitter(handle: "@Renox0")
+                Link(attributes: [
+                    HTMLAttribute(attribute: "rel", value: "stylesheet"),
+                    HTMLAttribute(
+                        attribute: "href",
+                        value: "https://cdnjs.cloudflare.com/ajax/libs/milligram/1.4.1/milligram.min.css"
+                    ),
+                    HTMLAttribute(
+                        attribute: "integrity",
+                        value: "sha512-xiunq9hpKsIcz42zt0o2vCo34xV0j6Ny8hgEy"
+                            + "lN3XBglZDtTZ2nwnqF/Z/TTCc18sGdvCjbFInNd++6q3J0N6g=="),
+                    HTMLAttribute(attribute: "crossorigin", value: "anonymous"),
+                ])
+                Script().source("/script.js")
+                Script { "connectToTheUserStoryVoteWebSocket()" }
+                Script { "preventFormSubmit('add-participant-form')" }
+            }
```

Comment:
All this part is constantly repeated in all templates. Would be good to have a function that avoid us repeating ourself.
Could be done in another PR (raise an issue).

Reply:
Done #28